paddle/framework/op_registry_test.cc

```diff
@@ -88,8 +89,7 @@ TEST(OpRegistry, IllegalAttr) {
 
   bool caught = false;
   try {
-    paddle::framework::OperatorBase* op __attribute__((unused)) =
-        paddle::framework::OpRegistry::CreateOp(op_desc);
+    OpPtr op __attribute__((unused)) = OpRegistry::CreateOp(op_desc);
```

Comment:
??
GTEST has a `ASSERT_THROW` macro.

Maybe in following PRs that part could be changed.